lib/client-logger/src/index.ts

```diff
@@ -1,16 +1,25 @@
-const { console } = global;
-
-/* tslint:disable: no-console */
+import { window, LOGLEVEL } from 'global';
+import console from 'loglevel';
 
 export const logger = {
+  trace: (message: any, ...rest: any[]): void => console.trace(message, ...rest),
   debug: (message: any, ...rest: any[]): void => console.debug(message, ...rest),
-  log: (message: any, ...rest: any[]): void => console.log(message, ...rest),
+  log: (message: any, ...rest: any[]): void => window.console.log(message, ...rest),
   info: (message: any, ...rest: any[]): void => console.info(message, ...rest),
   warn: (message: any, ...rest: any[]): void => console.warn(message, ...rest),
   error: (message: any, ...rest: any[]): void => console.error(message, ...rest),
 };
 
-export const pretty = (type: keyof typeof logger) => (...args: string[]) => {
+if (LOGLEVEL) {
+  console.setLevel(LOGLEVEL);
+}
+
+// eslint-disable-next-line no-underscore-dangle
+window.__STORYBOOK_LOGGER = console;
```

Comment:
Should we just check if `window` is defined before setting this?

Reply:
✅